lang/en.js

```diff
@@ -96,6 +96,7 @@ var language = {
       'unavailable': 'With WebUSB you can program your micro:bit and connect to the serial console directly from the online editor.<br/>Unfortunately, WebUSB is not supported in this browser. We recommend Chrome, or a Chrome-based browser to use WebUSB.',
       'find-more': 'Find Out More'
     },
+    'troubleshoot': 'Troubleshoot',
```

Comment:
@microbit-mark Will you roundtrip the translation in Crodwin?

Reply:
Yes, we should probably do this from v2-beta when merged

Reply:
We briefly discussed this in the stand-up today.
It sounds like when a new string is added the process is to:
- Developer adds string key-value entry in en.js and commits to the repo
- Developer round-trips in Crowdin en.js to export to the other languages
- Crodwin exports es.js and pl.js files with the new strings key-value entries
- Developer commits the es.js and pl.js files into the repo

If this is correct until we get the es.js and pl.js files from Crodwin then the website might break when the user changes the language, as the code expects the key-value entries to be present in the es.js and pl.js files, which it will not find.
For that reason I would prefer to do the round-tripping before merging into v2-beta or master, as those branches should be always fully 100% operational.